web/.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Drupal editor configuration normalization
```

Comment:
Is this file needed? Looks like it was duplicated from the directory above `/web`?

Reply:
Yep not sure how that happened - removed it